### PR TITLE
Helm chart

### PR DESCRIPTION
**Pull request by iffyuva**
Inspired by traefik helm chart and comments by @maikotz in this issue https://github.com/matomo-org/docker/issues/62

I have omitted 2 things here:
- Cron job for archiving data
- Matomo config as configMap

Let me know your thoughts.

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (pierreozoux) [inline helm/Chart.yaml]: description is not good.

File: helm/values.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  port: 80
+
+ingress:
+  enabled: false
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  path: /
+  hosts:
+    - analytics.yourdomain.com
```
Comment (pierreozoux) [inline helm/values.yaml]: Maybe we shouldn't have a default here.

**Comment by iffyuva** [comment]
@pierreozoux thanks for merging the PR. What do you mean by making the chart official? Push it here https://github.com/kubernetes/charts/ ?

**Comment by diesl** [comment]
Unfortunately it seems that this PR has broken the automation to bring a new version of Matomo to Docker Hub, see issue #115. Maybe you can help to fix this? Thanks!

**Pull request merged**